### PR TITLE
restructure quota mgmt docs

### PR DESCRIPTION
**Pull request by imatwawana**
Restructures Quota Management index page to be significantly shorter
Adds Stats image at top of page
Adds Quick Guide table
Moves content from index page to guides
Moves Does it Count table to end of page
Adds Attachments Quota guide

Merge in conjunction with [PR 36666](https://github.com/getsentry/sentry/pull/36666) to update in-app docs link.

### CODE REVIEW

**Comment by imatwawana** [comment]
FYI to reviewers, the files with the most changes don't load when automatically because the diff is big, so make sure you don't miss those files (namely the index page and the error quota mgmt page).

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx [hunk 1]
```diff
@@ -13,179 +13,306 @@ Sending all your application errors to Sentry ensures you'll be notified in real
...
+Spike protection is enabled for your organization by default, and when it's enabled, Sentry continually monitors for spikes. You can confirm that it's enabled in **Settings > Subscription**. This page can only be accessed by a Billing or Owner member of your Sentry organization.
 
-**PHP** - The `error_types` configuration option allows you to set the error types you want Sentry to monitor. Learn more in PHP [Basic Options](/platforms/php/configuration/options/#common-options).
+**We also include a one-time grace period for all paid accounts.** If an account depletes its event volume and has never previously triggered a grace period, we continue to accept events for the next three days at a maximum rate of 10,000 events per hour. If you choose not to increase your volume, Sentry will stop accepting events for the remainder of the current billing month. However, if you increase your volume, events accepted during the grace period will be counted as consumed volume.
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:52]: Does line 52 have anything to do with spike protection? It was with this content in the original so I haven't moved it, but it's not clear to me that it has any relation to spike protection. Can someone confirm? If not, then I'm going to move it.

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx [hunk 2]
```diff
@@ -13,179 +13,306 @@ Sending all your application errors to Sentry ensures you'll be notified in real
...
 
-**Ruby** - The `excluded_exceptions` configuration option allows you to set the exception types you wish to suppress. Learn more in the [Ruby configuration docs](/platforms/ruby/configuration/options/#optional-settings)
+<!-- does the paragraph above belong with spike protection or is this an entirely separate thing? does spike protection have to be enabled for the grace period to kick in?? -->
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:54]:
```suggestion
```

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx [hunk 3]
```diff
@@ -13,179 +13,306 @@ Sending all your application errors to Sentry ensures you'll be notified in real
...
+- If it's a specific release version that has caused the spike, add the version identifier to the project's [inbound filters](#3-inbound-data-filters) to avoid accepting events from that release.
+
+To review the error events dropped because of spike protection, go to the "Usage Stats" tab of **Stats** for your Sentry org and select "Errors" in the "Category" dropdown.
+
+### When Does Spike Protection Become "Inactive?"
+
+Events will not be dropped during any minute in which you don't send more than the hourly limit that Sentry has calculated for you. After 24 hours without any dropped events, spike protection becomes "inactive" again. This means that it is no longer dropping events, but _it does not mean the system has stopped paying attention._ It does however mean that the next time events are dropped, with respect to alerts it will count as a "reactivation" of spike protection.
+
+<!-- this last sentence sounds like a penalty of some sort; is it? or is just a fact? -->
+
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:107]:
```suggestion
```

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx [hunk 4]
```diff
@@ -13,179 +13,306 @@ Sending all your application errors to Sentry ensures you'll be notified in real
...
+- Set up an [on-demand budget](#on-demand-budget) to ensure you have time to adjust your volume in the event of a future spike in errors.
+- Set better [rate limits](#6-rate-limiting) on the DSN keys for the projects related to the spike.
+- If it's a specific release version that has caused the spike, add the version identifier to the project's [inbound filters](#3-inbound-data-filters) to avoid accepting events from that release.
+
+To review the error events dropped because of spike protection, go to the "Usage Stats" tab of **Stats** for your Sentry org and select "Errors" in the "Category" dropdown.
+
+### When Does Spike Protection Become "Inactive?"
+
+Events will not be dropped during any minute in which you don't send more than the hourly limit that Sentry has calculated for you. After 24 hours without any dropped events, spike protection becomes "inactive" again. This means that it is no longer dropping events, but _it does not mean the system has stopped paying attention._ It does however mean that the next time events are dropped, with respect to alerts it will count as a "reactivation" of spike protection.
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:104]: This last sentence sounds like a penalty of some sort; is it? Or is just a fact?

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx (hunk 1 above)
Comment (souredoutlook) [reply src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:52]: Grace period almost needs its own section. It's not related to Spike Protection however the two questions are often related for customers that experiencing large surges of events that deplete their w

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx (hunk 4 above)
Comment (souredoutlook) [reply src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:103]: You could rephrase this to say that the next time spike protection is activated you will be alerted. There are a lot of statements in the negative in this block which makes the content hard to follow:

> Events will be dropped during any minute in which you send more than the hourly limit that Sentry has calculated for you. 

Feels easier to follow.

File: src/docs/product/accounts/quotas/index.mdx [hunk 5]
```diff
@@ -7,317 +7,116 @@ redirect_from:
...
+### 4. Event Repetition
 
-<Note>
+In some cases, repeated events can count against your quota, so it's important to know how event repetition is treated in Sentry. Learn more in [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#5-applying-workflows).
 
-The 24-hour window ends at the beginning of the current hour, not at the current minute.
+### 5. Inbound Filters {#inbound-data-filters}
 
-</Note>
+If an inbound filter is set for a type of error, transaction, or attachment, and your subscription allows, it won't be counted. You can set these in **[Project] > Settings > Inbound Filters**. Learn more in [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#3-inbound-data-filters), [Manage Your Transaction Quota](/product/accounts/quotas/manage-transaction-quota/#2-inbound-data-filters), and [Manage Your Attachments Quota](/product/accounts/quotas/manage-attachments-quota/#3-inbound-data-filters).
```
Comment (souredoutlook) [inline src/docs/product/accounts/quotas/index.mdx:79]: Consider "If an inbound filter is applied to an error, ..." Inbound filters can't be set for specific transactions or attachments but some inbound filters do apply to all data categories.

File: src/docs/product/accounts/quotas/manage-transaction-quota.mdx [hunk 6]
```diff
@@ -2,51 +2,157 @@
...
+
+![Usage Stats page showing transactions](usage-stats-transactions.png)
+
+### What are my busiest projects?
+
+The "Project" table in the "Usage Stats" tab of **Stats** breaks down your events by project so you can see which ones are consuming your quota. Clicking on the settings icon next to a project name in the table will open the project's settings, where you can manage its [inbound filters](#2-inbound-data-filters).
+
+### Which transactions are consuming my quota?
+
+You can set up a query in **Discover** to see which transactions you're receiving the most and if you want to adjust that. When you're building the query, search for `event.type:transaction` and then set the columns to `title`, `project`, and `count`, as shown below:
```
Comment (souredoutlook) [inline src/docs/product/accounts/quotas/manage-transaction-quota.mdx:30]: Is it worth setting up a warning here to indicate that this search is only possible on Business plan? You could also create this query and use org redirects so that customers could easily navigate right to the page with the query preselected like so:

https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count()&name=All+Events&query=event.type%3Atransaction&sort=-title&statsPeriod=14d&yAxis=count()

File: src/docs/product/accounts/quotas/manage-transaction-quota.mdx [hunk 7]
```diff
@@ -2,51 +2,157 @@
...
+
+When you exceed your quota threshold, the server will respond with a 429 HTTP status code, which communicates to SDKs and clients to stop sending events. This status code comes with a `Retry-After` header that indicates the time for which this rate limit is active. However, clients are not supposed to retry events, but instead drop events until the rate limit has expired, to prevent queue backlogs. Note, that since event ingestion and rate limiting happen asynchronously, the 429 HTTP status code is always slightly delayed.
+
+To increase your quota, go to **Settings > Subscription** and click the "Manage Subscription" button to access your subscription options. When you increase your quota, the change goes into effect immediately.
+
+If you're on a Developer plan and want to increase your quota, you'll need to upgrade to a Team or Business plan. On these plans you can prepay for more event volume and purchase on-demand volume, as needed. Learn about Sentry's plans on our [pricing page](https://sentry.io/pricing/).
+
+<Note>
```
Comment (souredoutlook) [inline src/docs/product/accounts/quotas/manage-transaction-quota.mdx:60]: Should this be a warning instead of a note? This causes a fair bit of pain when it does happen.

File: src/docs/product/accounts/quotas/index.mdx [hunk 8]
```diff
@@ -7,317 +7,116 @@ redirect_from:
...
 
-You can view and restore previously discarded issues by navigating to the "Discarded Issues" tab of **[Project] > Settings > Inbound Filters**.
+- **Event** - An event is one instance of you sending data to Sentry. Generally, this data is an error or a transaction.
+- **Error** - What counts as an error varies by platform, but in general, if there's something that looks like an exception, it can be captured as an error in Sentry. Sentry automatically captures errors, uncaught exceptions, and unhandled rejections, as well as other types of errors, depending on platform. A grouping of similar errors makes [an issue](/product/issues/).
+- **Transaction** - A [transaction](/product/performance/transaction-summary/#what-is-a-transaction) represents a single instance of a service being called to support an operation you want to measure or track, like a page load. Transaction events are grouped by the transaction name.
+- **Attachment** - Attachments are stored additional files, such as config or log files that are attached to an error event. Unless the option to store crash reports is enabled, Sentry will use these files only to create an event, and then drop them.
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply src/docs/product/accounts/quotas/index.mdx:51]: > Is it worth talking about stored mini-dumps here?

Like as in defining stored mini-dumps or just mentioning them as a type of attachment?

File: src/docs/product/accounts/quotas/index.mdx [hunk 9]
```diff
@@ -7,317 +7,116 @@ redirect_from:
...
 
-To set up rate limits, navigate to **[Project] > Settings > Client Keys** and click "Configure"\*\*". Select an individual key or create a new one, then you’ll be able to define a rate limit as well as view a breakdown of events received by that key. For additional information and examples, see [Rate Limiting in our guide to managing your error quota](/product/accounts/quotas/manage-event-stream-guide/#6-rate-limiting).
+<!-- prettier-ignore-start -->
 
-<Note>
+| Action | Errors | Transactions | Attachments |
+| ------ | ------ | ------------ | ----------- |
+| 1. [Check your spike protection is enabled](#spike-protection) | &check; |  |  |
+| 2. [Adjust your quota](#increasing-quotas) | &check; | &check; | &check; |
+| 3. [Rate limit your events](#limiting-events) | &check; |  | &check; |
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply src/docs/product/accounts/quotas/index.mdx:34]: > Does rate limiting actually prevent storage of attachments? I believe this is not the case.

We use the term "rate limit" for the two different actions in the original docs so I followed what we had originally, (see the Rate Limits section on this page). But if it's not clear, then it probably makes sense to split these apart.

File: src/docs/product/accounts/quotas/index.mdx [hunk 10]
```diff
@@ -7,317 +7,116 @@ redirect_from:
...
+| 4. [Review repeated events](#4-event-repetition) | &check; |  |  |
+| 5. [Filter your events](#inbound-data-filters) | &check; | &check; | &check; |
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply src/docs/product/accounts/quotas/index.mdx:36]: > https://getsentry.atlassian.net/browse/ISSUE-1339

Do you mean add something in the longer explanation (not here) indicating that attachments for filtered events might still be stored and this is a known issue?

File: src/docs/product/accounts/quotas/index.mdx (hunk 5 above)
Comment (imatwawana) [reply src/docs/product/accounts/quotas/index.mdx:79]: > Consider "If an inbound filter is applied to an error, ..." Inbound filters can't be set for specific transactions or attachments but some inbound filters do apply to all data categories.

Do you know which ones apply in which cases? I just updated that content recently from saying inbound filters don't apply to transactions and attachments at all to saying it does and I wasn't given any indication that it was specific.

File: src/docs/product/accounts/quotas/index.mdx (hunk 9 above)
[the remark this answers is not in the document]
Comment (souredoutlook) [reply src/docs/product/accounts/quotas/index.mdx:34]: I think I reference this in another comment but I believe rate-limited error events with separate attachments will still store the attachment.

File: src/docs/product/accounts/quotas/index.mdx (hunk 5 above)
Comment (souredoutlook) [reply src/docs/product/accounts/quotas/index.mdx:79]: IP filters should filter all event types. As does filtering by release I believe. You might want to verify that with the Ingest team though.

File: src/docs/product/accounts/quotas/index.mdx (hunk 9 above)
[the remark this answers is not in the document]
Comment (souredoutlook) [reply src/docs/product/accounts/quotas/index.mdx:34]: I believe some attachments are included in error event envelopes now.

Comment (imatwawana) [inline src/docs/product/accounts/quotas/index.mdx:34]:
```suggestion
| 3. [Rate limit your events or attachments](#limiting-events) | &check; |  | &check; |
```

File: src/docs/product/accounts/quotas/index.mdx (hunk 8 above)
Comment (imatwawana) [inline src/docs/product/accounts/quotas/index.mdx:51]:
```suggestion
- **Attachment** - Attachments are stored additional files, such as config files, log files, or stored mini-dumps that are attached to an error event. Unless the option to store crash reports is enabled, Sentry will use these files only to create an event, and then drop them.
```

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx [hunk 11]
```diff
@@ -13,179 +13,303 @@ Sending all your application errors to Sentry ensures you'll be notified in real
...
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:103]:
```suggestion
Events will not be dropped during any minute in which you don't send more than the hourly limit that Sentry has calculated for you. After 24 hours without any dropped events, spike protection becomes "inactive" again. This means that it is no longer dropping events, but _it does not mean the system has stopped paying attention._ The next time events are dropped, spike protection will be "reactivated".
```

File: src/docs/product/accounts/quotas/manage-transaction-quota.mdx (hunk 7 above)
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-transaction-quota.mdx:60]:
```suggestion
<Alert>
```

File: src/docs/product/accounts/quotas/manage-transaction-quota.mdx [hunk 12]
```diff
@@ -2,51 +2,157 @@
...
+
+If you upgrade from Team to Business mid-billing period, your on-demand pricing changes retroactively.
+
+</Note>
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-transaction-quota.mdx:64]:
```suggestion
</Alert>
```

File: src/docs/product/accounts/quotas/manage-transaction-quota.mdx [hunk 13]
```diff
@@ -2,51 +2,146 @@
...
+The [Usage Stats](/product/stats/#usage-stats) tab displays details about the total amount of data Sentry has received across your entire organization for up to 90 days. The page breaks down the events (by project) into three categories: _accepted_, _dropped_, or _filtered_. Only accepted events affect your quota:
+
+![Usage Stats page showing transactions](usage-stats-transactions.png)
+
+### What are my busiest projects?
+
+The "Project" table in the "Usage Stats" tab of **Stats** breaks down your events by project so you can see which ones are consuming your quota. Clicking on the settings icon next to a project name in the table will open the project's settings, where you can manage its [inbound filters](#2-inbound-data-filters).
+
+### Which transactions are consuming my quota?
+
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-transaction-quota.mdx:29]:
```suggestion

<Note>

You can only update the columns of the Results table in **Discover** if your organization is on a Business plan.

</Note>

```

**Review by sentrypri — CHANGES_REQUESTED** [review]
Line 16: This page is accessible to all members of your organization, so a Billing or Owner member can share this page with the developers directly responsible for a given project.

I'd remove the reference to billing members. While the stats tab is available for billing members to click on, the data doesn't actually load for them and they'll get an error saying they don't have enough permission to view the data or something like that.

File: src/docs/product/accounts/quotas/index.mdx [hunk 14]
```diff
@@ -7,317 +7,116 @@ redirect_from:
...
+This table outlines whether or not an an event or attachment that you send to Sentry counts against your quota.
 
-Sentry imposes limits on various fields within an event, as well as the size of full events and the requests they are sent in:
+<!-- prettier-ignore-start -->
 
-- Events, attachments, and requests exceeding payload size limits are immediately dropped with a `413 Payload Too Large` error. Sentry allows compressed content encoding, and applies separate limits before and after decompression.
-- Fields exceeding the individual size limits are afterwards trimmed and truncated at a best effort.
+| Scenario | Yes, this data counts |
+|----------|------------------------|
+| [Spike protection](#spike-protection) has been triggered (errors) |  |
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/index.mdx:105]:
```suggestion
| [Spike protection](#spike-protection) activation threshold has been reached (errors) |  |
| [Spike protection](#/product/accounts/quotas/manage-event-stream-guide/#when-does-spike-protection-become-inactive) no longer active (errors) | &check; |
```

File: src/docs/product/accounts/quotas/manage-attachments-quota.mdx [hunk 15]
```diff
@@ -0,0 +1,118 @@
...
+
+Applying the proper filters, SDK configuration, and rate limits is an iterative and on-going process, but these tips will show you how to get the most out of your error events. These tips are ordered from easiest or least time-consuming to most challenging or potentially time-consuming. The first three options are all things you can do in [sentry.io](https://sentry.io), whereas the remaining one is something you'll do in the SDK.
+
+## Before You Begin: Check Your Quota Usage
+
+You can look at your attachments in aggregate in the "Usage Stats" tab of **Stats**. This information will help you answer key questions about the breakdown of your incoming attachments or which projects are consuming your quota. The answers to these questions can help you figure out where you need to do further fine-tuning of your SDK filters and configuration.
+
+This page is accessible to all members of your organization, so a Billing or Owner member can share this page with the developers directly responsible for a given project. Also, you can come back to this page to assess if the changes you've made are having the desired effect.
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-attachments-quota.mdx:16]:
```suggestion
This page is accessible to all members of your organization, so Sentry org Owners can share this page with the developers directly responsible for a given project. Also, you can come back to this page to assess if the changes you've made are having the desired effect.
```

File: src/docs/product/accounts/quotas/manage-transaction-quota.mdx [hunk 16]
```diff
@@ -2,51 +2,152 @@
...
 
-When you configure your the SDK, you can control the number of transactions that are sent to Sentry by setting the [tracing options](/platform-redirect/?next=/configuration/options/%23tracing-options). You can also set up [custom instrumentation](/platform-redirect/?next=/performance/instrumentation/custom-instrumentation/) for performance monitoring to capture certain types of transactions.
+## Before You Begin: Check Your Quota Usage {#3-event-usage-stats}
+
+You can look at your events in aggregate in the "Usage Stats" tab of **Stats**. This information will help you answer key questions about the breakdown of your incoming events or which projects are consuming your quota. The answers to these questions can help you figure out where you need to do further fine-tuning of your SDK filters and configuration.
+
+This page is accessible to all members of your organization, so a Billing or Owner member in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-transaction-quota.mdx:16]:
```suggestion
This page is accessible to all members of your organization, so Owners in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.
```

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx [hunk 17]
```diff
@@ -13,179 +13,312 @@ Sending all your application errors to Sentry ensures you'll be notified in real
...
+You can look at your events in aggregate in the "Usage Stats" tab of **Stats**. This information will help you answer key questions about the breakdown of your incoming events or which projects are consuming your quota. The answers to these questions can help you figure out where you need to do further fine-tuning of your SDK filters and configuration.
 
-## 2. SDK Configuration
+This page is accessible to all members of your organization, so a Billing or Owner member in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:22]:
```suggestion
This page is accessible to all members of your organization, so Owners in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.
```

File: src/docs/product/accounts/quotas/manage-attachments-quota.mdx [hunk 18]
```diff
@@ -0,0 +1,118 @@
...
+
+- Events, attachments, and requests exceeding payload size limits are immediately dropped with a `413 Payload Too Large` error. Sentry allows compressed content encoding, and applies separate limits before and after decompression.
+- Fields exceeding the individual size limits are afterwards trimmed and truncated at a best effort.
+
+To avoid using up your attachments quota — whch is based on size and not number of instaces of attachments data sent — or having the event dropped entirely, consider limiting the size of values passed into Sentry's APIs. For example, if your application attaches application state or request bodies to Sentry events, truncate them first.
+
+The precise limits may change over time. For more information, please refer to the following resources:
+
+- [Envelope Size Limits](https://develop.sentry.dev/sdk/store/#size-limits)
+- [Store Endpoint Size Limits](https://develop.sentry.dev/sdk/store/#size-limits)
```
Comment (wmak) [inline src/docs/product/accounts/quotas/manage-attachments-quota.mdx:110]: Are these two links intentionally the same?

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx [hunk 19]
```diff
@@ -13,179 +13,312 @@ Sending all your application errors to Sentry ensures you'll be notified in real
...
+You can set up [a query](https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count%28%29&name=All+Events&query=event.type%3Aerror&sort=-title&statsPeriod=14d&yAxis=count%28%29) in **Discover** to take a more proactive approach to resolving your busiest issues. When you're building the query, search for `event.type:error` and then set the columns `issue`, `title`, `project`, and `count()`, as shown below:
 
-Check out additional configuration options with the [tryCatch](/platforms/javascript/configuration/integrations/default/#trycatch) and [ReportingObserver](/platforms/javascript/configuration/integrations/plugin/#reportingobserver) integrations.
+![Busiest Projects](manage-event-stream-02.png)
 
-### Other SDKs
+Once the changes are applied, sort the "Results" table by the "COUNT" column to display your busiest issues:
```
Comment (wmak) [inline src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:46]:
```suggestion
Once the changes are applied, sort the "Results" table by the "count()" column to display your busiest issues:
```
or if you want to match screenshot
```suggestion
Once the changes are applied, sort the "Results" table by the "COUNT()" column to display your busiest issues:
```

File: src/docs/product/accounts/quotas/manage-transaction-quota.mdx [hunk 20]
```diff
@@ -2,51 +2,152 @@
...
+
+### Which transactions are consuming my quota?
+
+<Note>
+
+You can only update the columns of the Results table in **Discover** if your organization is on a Business plan.
+
+</Note>
+
+You can set up [a query](<https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count()&name=All+Events&query=event.type%3Atransaction&sort=-title&statsPeriod=14d&yAxis=count()>) in **Discover** to see which transactions you're receiving the most and if you want to adjust that. When you're building the query, search for `event.type:transaction` and then set the columns to `title`, `project`, and `count`, as shown below:
```
Comment (wmak) [inline src/docs/product/accounts/quotas/manage-transaction-quota.mdx:36]:
```suggestion
You can set up [a query](<https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count()&name=All+Events&query=event.type%3Atransaction&sort=-count&statsPeriod=14d&yAxis=count()>) in **Discover** to see which transactions you're receiving the most and if you want to adjust that. When you're building the query, search for `event.type:transaction` and then set the columns to `title`, `project`, and `count`, as shown below:
```

File: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx [hunk 21]
```diff
@@ -13,179 +13,312 @@ Sending all your application errors to Sentry ensures you'll be notified in real
...
 
-- [Filtering](/platforms/javascript/configuration/filtering/)
-- [Enriching Events: Add Context](/platforms/javascript/enriching-events/context/)
+You can only update the columns of the Results table in **Discover** if your organization is on a Business plan.
 
-#### GlobalHandlers
+</Note>
 
-The integration attaches global handlers to capture uncaught exceptions (`onerror`) and unhandled rejections (`onunhandledrejection`). Both handlers are enabled by default, but can be disabled through configuration. Learn more in [GlobalHandlers Integration](/platforms/javascript/configuration/integrations/default/#globalhandlers).
+You can set up [a query](https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count%28%29&name=All+Events&query=event.type%3Aerror&sort=-title&statsPeriod=14d&yAxis=count%28%29) in **Discover** to take a more proactive approach to resolving your busiest issues. When you're building the query, search for `event.type:error` and then set the columns `issue`, `title`, `project`, and `count()`, as shown below:
```
Comment (wmak) [inline src/docs/product/accounts/quotas/manage-event-stream-guide.mdx:42]:
```suggestion
You can set up [a query](https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count%28%29&name=All+Events&query=event.type%3Aerror&sort=-count&statsPeriod=14d&yAxis=count%28%29) in **Discover** to take a more proactive approach to resolving your busiest issues. When you're building the query, search for `event.type:error` and then set the columns `issue`, `title`, `project`, and `count()`, as shown below:
```

File: src/docs/product/accounts/quotas/manage-transaction-quota.mdx [hunk 22]
```diff
@@ -2,51 +2,152 @@
...
+
+![Busiest Projects](manage-trans-quota-01.png)
+
+Once the changes are applied, sort the "Results" table by the "COUNT" column to display your busiest transactions:
```
Comment (wmak) [inline src/docs/product/accounts/quotas/manage-transaction-quota.mdx:40]:
```suggestion
Once the changes are applied, sort the "Results" table by the "COUNT()" column to display your busiest transactions:
```

**Review by wmak — APPROVED** [review]
Mostly minor comments, overall LGTM

File: src/docs/product/accounts/quotas/index.mdx [hunk 23]
```diff
@@ -7,317 +7,117 @@ redirect_from:
...
-## Limiting Events
+You can also download a project breakdown report in the "Usage History" tab of **Settings > Subscriptions** (only accessible to Owner and Billing members of your Sentry organization).
 
-You can add limits for error events and attachments in [sentry.io](https://sentry.io). This does not apply to transaction events.
+Sentry’s flexibility means you can exercise fine-grained control over which events and attachments count toward your quota. This page provides you with high-level information about strategies for managing your quota, but you can get more detailed information in:
 
-### Error Event Limits
+- [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/)
+- [Manage Your Transaction Quota](/product/accounts/quotas/manage-transaction-quota/)
+- [Manage Your Attachments Quota](/product/accounts/quotas/manage-attachments-quota/)
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply src/docs/product/accounts/quotas/index.mdx:22]: Yes, 'cause events includes transactions. We spell that out further up the page.

File: src/docs/product/accounts/quotas/index.mdx [hunk 24]
```diff
@@ -7,317 +7,117 @@ redirect_from:
...
+- **Transaction** - A [transaction](/product/performance/transaction-summary/#what-is-a-transaction) represents a single instance of a service being called to support an operation you want to measure or track, like a page load. Transaction events are grouped by the transaction name.
+- **Attachment** - Attachments are stored additional files, such as config files, log files, or stored mini-dumps that are attached to an error event. Unless the option to store crash reports is enabled, Sentry will use these files only to create an event, and then drop them.
+- **Data** - Anything you send to Sentry. This includes, events (errors or transactions), attachments, and event metadata.
+- **Quota** - Your quota is the amount of data (events, transactions, attachments) that you pay Sentry to track.
 
-## Spike Protection
+## Manage Your Quota - Overview {#what-counts-toward-my-quota-an-overview}
 
-<Alert level="info">
+Sentry completes a thorough evaluation of each event to determine if it counts toward your quota, as outlined in this overview. Detailed documentation for each evaluation is linked throughout.
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply src/docs/product/accounts/quotas/index.mdx:57]: Good call. I've added a note after the second paragraph

File: src/docs/product/accounts/quotas/index.mdx [hunk 25]
```diff
@@ -7,317 +7,117 @@ redirect_from:
...
 
-Spike protection is not currently available for transactions nor attachments.
+Before completing any of these evaluations, Sentry confirms that each event includes a valid DSN and project as well as whether the event can be parsed. In addition, for error events, Sentry validates that the event contains valid fingerprint information. If any of these items are missing or incorrect, the event is rejected.
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/index.mdx:59]:
```suggestion
Before completing any of these evaluations, Sentry confirms that each event includes a valid DSN and project as well as whether the event can be parsed. In addition, for error events, Sentry validates that the event contains valid fingerprint information. If any of these items are missing or incorrect, the event is rejected.

<Note>

This list is ordered from easiest or least time-consuming to most challenging or potentially time-consuming. This isn't the order in which evaluations take place. 

</Note>
```

File: src/docs/product/accounts/quotas/manage-attachments-quota.mdx (hunk 18 above)
Comment (imatwawana) [reply src/docs/product/accounts/quotas/manage-attachments-quota.mdx:110]: Good catch! Fixed

File: src/docs/product/accounts/quotas/manage-attachments-quota.mdx [hunk 26]
```diff
@@ -0,0 +1,118 @@
...
+Sentry imposes limits on various fields within an event, as well as the size of full events and the requests they are sent in:
+
+- Events, attachments, and requests exceeding payload size limits are immediately dropped with a `413 Payload Too Large` error. Sentry allows compressed content encoding, and applies separate limits before and after decompression.
+- Fields exceeding the individual size limits are afterwards trimmed and truncated at a best effort.
+
+To avoid using up your attachments quota — whch is based on size and not number of instaces of attachments data sent — or having the event dropped entirely, consider limiting the size of values passed into Sentry's APIs. For example, if your application attaches application state or request bodies to Sentry events, truncate them first.
+
+The precise limits may change over time. For more information, please refer to the following resources:
+
+- [Envelope Size Limits](https://develop.sentry.dev/sdk/store/#size-limits)
```
Comment (imatwawana) [inline src/docs/product/accounts/quotas/manage-attachments-quota.mdx:109]:
```suggestion
- [Envelope Size Limits](https://develop.sentry.dev/sdk/envelopes/#size-limits)
```
